tuf/client/updater.py

```diff
@@ -3241,6 +3247,30 @@ def download_target(self, target, destination_directory,
         to ensure snapshot consistency).
         Default is True.
 
+     custom_download_handler:
+        A user provided function performing the actual target file download.
+        In order to comply with the TUF specification, the function signature
+        should have the form:
+
+        'def download_handler_func(url, required_length)'
+
+        The function implementation should provide the following functionality:
+
+        'Given the 'url' and 'required_length' of the desired file, open a
+        connection to 'url', download it, and return the contents of the file.
```

Comment:
I think "contents of the file" means a file object, that should be explicit. It might even be useful to mention that it really should be a TemporaryFile() -- nothing else really makes sense.

Maybe makes sense to document the handler in the same way TUF functions are documented (even if the format is a bit weird)?

Reply:
I updated the docstring as you suggest (aa3a635 ). The text is copied from the current `tuf.download` module documentation and I don't insist on this content. It is what currently TUF claims doing.